modules/dnn/src/tensorflow/tf_importer.cpp

```diff
@@ -1469,11 +1469,29 @@ void TFImporter::populateNet(Net dstNet)
 
             layerParams.set("height", outSize.at<int>(0, 0));
             layerParams.set("width", outSize.at<int>(0, 1));
+            layerParams.set("interpolation", 0);
 
             if (hasLayerAttr(layer, "align_corners"))
                 layerParams.set("align_corners", getLayerAttr(layer, "align_corners").b());
 
-            int id = dstNet.addLayer(name, "ResizeNearestNeighbor", layerParams);
+            int id = dstNet.addLayer(name, "Resize", layerParams);
+            layer_id[name] = id;
+
+            connect(layer_id, dstNet, parsePin(layer.input(0)), id, 0);
+        }
+        else if (type == "ResizeBilinear")
```

Comment:
Body of that case is absolutely the same. Can you just extend the previous one by `else if (type == "ResizeNearestNeighbor" || type == "ResizeBilinear")` and vary "interpolation" flag?